src/controllers/user.js

```diff
@@ -15,14 +16,14 @@ export const create = async (req, res) => {
     const existingUser = await User.findByEmail(userToCreate.email)
 
     if (existingUser) {
-      return sendDataResponse(res, 400, { error: 'Email already in use' })
+      return sendDataResponse(res, 400, { error: ERR.EMAIL_IN_USE })
     }
 
     const createdUser = await userToCreate.save()
 
     return sendDataResponse(res, 201, createdUser)
   } catch (error) {
-    return sendMessageResponse(res, 500, 'Unable to create new user')
+    return sendMessageResponse(res, 500, ERR.UNABLE_TO_CREATE_USER)
```

Comment:
can we add a console error to the catch block

Reply:
Let's add some context to it as well like "error creating user"

Reply:
`console.error()`